shopinvader_api_sale/routers/sale_lines.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2024 Akretion (http://www.akretion.com).
+# @author Florian Mounier <florian.mounier@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from typing import Annotated
+
+from fastapi import APIRouter, Depends
+
+from odoo import api, fields, models
+
+from odoo.addons.base.models.res_partner import Partner as ResPartner
+from odoo.addons.extendable_fastapi.schemas import PagedCollection
+from odoo.addons.fastapi.dependencies import (
+    authenticated_partner,
+    authenticated_partner_env,
+    paging,
+)
+from odoo.addons.fastapi.schemas import Paging
+from odoo.addons.sale.models.sale_order_line import SaleOrderLine
+from odoo.addons.shopinvader_filtered_model.utils import FilteredModelAdapter
+
+from ..schemas import SaleLineSearch, SaleLineWithSale
+
+sale_line_router = APIRouter(tags=["sales"])
+
+
+@sale_line_router.get("/sale_lines")
```

Comment:
A more logic routing would be:

* /sales/lines -> all lines
* /sales/lines/{sale_line_id} -> specific line
* /sales/{sale_id}/lines -> lines of a specific sale order.